### PR TITLE
Dockerfile for mailer

### PR DESCRIPTION
**Pull request by apolkingg8**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+ARG NODEJS_VER=14
+ARG NPM_VER=7
+ARG TINI_VER=0.19
+
+ENV NODEJS_VER=$NODEJS_VER
+ENV NPM_VER=$NPM_VER
+ENV TINI_VER=$TINI_VER
+
+RUN apk add --no-cache nodejs~="$NODEJS_VER" tini~="$TINI_VER"
+RUN apk add --no-cache curl~=7 npm~="$NPM_VER"
```
Comment (solomondefi-dev) [inline Dockerfile:14]: Do we need npm, since you're getting pnpm directly?

**Review by solomondefi-dev — COMMENTED** [review]
I see this is still marked as draft, but I briefly looked through and left one comment.

As part of this work, can you uncomment the `build-docker-db-api:` in the `pr-commit.yaml` workflow, and change it so it builds the mailer Docker image?

**Comment by apolkingg8** [comment]
I think maybe we should reduce the image size by move some packages to peerDependence. Currently, the image is about 
1 GB, that's a big size.

File: Dockerfile (hunk 1 above)
Comment (apolkingg8) [reply Dockerfile]: Removed!

**Comment by solomondefi-dev** [comment]
Don't worry! I have a good solution for build size, I will add it after initial skaffold setup. The tracking issue is https://github.com/SolomonDefi/solomon-monorepo/issues/25

**Comment by solomondefi-dev** [comment]
I reviewed this yesterday and it seems fine to me. Is it ready to merge, or would you like to continue environment work @apolkingg8 ?

**Comment by apolkingg8** [comment]
Yes, it's ready to merge. I just reset the commit about the environment, will merge it after the test passes.

**Pull request merged**